include/librealsense2/dds/topics/control/control.idl

```diff
@@ -0,0 +1,24 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+module librealsense
+{
+    module dds
+    {
+        module topics
+        {
+            module raw
+            {
+                module device
+                {
+                    struct control
+                    {
+                        unsigned short id; // See "librealsense::dds::topics::control::msg_type"
```

Comment:
Do we really want to stick to enums to convey what we want?
Image a client and server that now need to be in sync. The client adds a message and therefore has to change an enum. But it has to add it at the end so it may be "out of order", so the server meanings aren't bad.

Why not instead have the "id" be a string? It's more readable, and I don't think has many drawbacks...

Reply:
Will be discussed in the meeting, not for current PR

Reply:
Will be changed, as discussed, in a different PR